src/protagonist/API/Features/Image/AssetBeforeProcessing.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using DLCS.HydraModel;
+using NuGet.ContentModel;
+
+namespace API.Features.Image;
+
+public class AssetBeforeProcessing
+{
+    public DLCS.Model.Assets.Asset Asset { get; init; } = null!;
+
+    public DeliveryChannel[]? DeliveryChannels { get; init; }
```

Comment:
this might be a little difficult to fix - in order for the delivery channels to be correctly converted, it requires calls to the database - decided the better option was to pass it through to the `AssetProcessor`, which has DB logic

Reply:
```cs
public record DeliveryChannelBeforeProcessing(string? Channel, string? Policy);
```